src/components/widget_types/twitter.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div>
+        <v-card v-for="tweet in tweets.slice(0, numberTweets)" v-bind:key="tweet.text">
+            <v-card-title>
+                <v-icon
+                    left
+                >
+                    mdi-twitter
+                </v-icon>
+                <span class="title font-weight-light">Twitter</span>
+            </v-card-title>
+
+            <v-card-text class="font-weight-bold">
+                {{tweet.text}}
+            </v-card-text>
+
+            <v-card-actions>
+                <v-list-item class="grow">
+                    <v-list-item-avatar color="grey darken-3">
+                    <v-img
+                        class="elevation-6"
+                        src="https://static.nd.edu/images/webclips/default/webclip-1024.png"
```

Comment:
The src should probably be a part of the tweets data structure, not hard-coded.

Reply:
added an imgSrc field to each config object, this line references that now